tests/test_prettytable.py

```diff
@@ -94,6 +94,60 @@ def mix_prettytable():
     return mix
 
 
+class TestNoneOption:
+    def test_none_char_valid_option(self):
+        t = PrettyTable(["Field 1", "Field 2", "Field 3"], none_format="")
```

Comment:
This `t` variable isn't used, so we can skip it (also silences the lint CI): 

```suggestion
        PrettyTable(["Field 1", "Field 2", "Field 3"], none_format="")
```